gateway/src/apicast/cli/command/start.lua

```diff
@@ -179,7 +179,7 @@ local function configure(cmd)
     cmd:flag("-d --daemon", "Daemonize.")
     cmd:option("-w --workers",
         "Number of worker processes to start.",
-        resty_env.value('APICAST_WORKERS') or 1)
+        resty_env.value('APICAST_WORKERS') or Environment.default_config.worker_processes)
```

Comment:
Should we call `tonumber` on the first part of the or? I guess it would work anyway, but I think it would be clearer.

Reply:
But what if you want to set `APICAST_WORKERS=auto` ?

Reply:
:+1:

Reply:
I think it's not obvious that APICAST_WORKERS accepts 'auto'. People familiar with nginx might guess it, but I think we should document it.

Reply:
`APICAST_WORKERS` is currently undocumented and I think it is pretty low level and stay like that for now. 

* in production we auto detect correct number of cores when deployed by our docker image
* development starts on 1 core

So people really should not need tuning the number of workers.